serverlessworkflow/modules/ROOT/pages/cloud/quarkus/deploying-on-openshift.adoc

```diff
@@ -0,0 +1,134 @@
+====
+If you are running OpenShift Local on Mac with M1 processors, you might not find the OpenShift Serverless Operator available.
+====
+
+Before proceeding further, make sure that you have access to the OpenShift cluster with Serverless available and the OpenShift Serverless operator is properly installed. To do this, please follow these link:{ocp_swf_install_url}[steps].
+
+
+[TIP]
+====
+If you get error messages related to `Istio`, this link:{knative_istio_issue_url}[article] might be helpful.
+====
+
+
+[[proc-deploy-sw-application-openshift]]
+== Deploying your workflow application on OpenShift
+
+Once `Knative Serving` is ready, you can initiate the process of deploying your {product_name} application on OpenShift.
+
+// shared app req
+include::../common/_deploy_workflow_application_requisites.adoc[]
+
+After checking the prerequisites, prepare the project that will be used to deploy your application
+.
+.Create project
+[source,shell,subs="attributes+"]
+----
+{command_line_tool} create project {default_namespace}
+----
+--
+
+. Set the Kubernetes context to the newly created namespace using the following command:
++
+--
+.Set OpenShift current project
+[source,shell]
+----
+{command_line_tool} project {default_namespace}
+----
+
+[NOTE]
+====
+You can use the link:build-workflow-image-with-quarkus-cli.html#proc-building-serverless-workflow-application-using-native-image[native image] due to the faster startup.
+====
+
+
+=== Preparing the OpenShift's Registry
+OpenShift comes with a registry where you can push your images built locally to be deployed.
+
+First, let's check if the Registry is already exposed, to do that, check if the route is exposed:
+[source,shell]
+----
+OPENSHIFT_REGISTRY_URL=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
+----
+[IMPORTANT]
+====
+The `OPENSHIFT_REGISTRY_URL` environment variable will be used in the next sections.
+====
+
+This command should assign to the `OPENSHIFT_REGISTRY_URL` variable the publicly accessible URL for the registry. If it does not return it, it does mean that it is not exposed, to expose the registry follow link:{ocp_expose_registry_url}[these] steps.
+it does mean that it is not exposed, to expose the registry follow link:{ocp_expose_registry_url}[these] steps.
+
+The OpenShift's registry requires you to be logged in to be allowed to push Container Images. If you are already logged in the cluster you can use your token to authenticate in the OpenShift's registry. To log in, save the URL from the command above:
+
+.Log in to the OpenShift's registry
+[source,shell]
+----
+podman login -u <USERNAME> -p $(oc whoami -t) --tls-verify=false $OPENSHIFT_REGISTRY_URL
+----
+If everything is working, you should get the *Login Succeeded!* message.
+
+The next step is to deploy your workflow application and execute it.
+You can read the further the next sections which explain different approaches deploy your {product_name} application, as such:
+
```

Comment:
In the included parts, we write about the `image-registry.openshift-image-registry.svc:5000` but we never explain where it does come from and this could be confusing for the user.